lisa/tools/docker.py

```diff
@@ -47,7 +47,7 @@ def run_container(
         docker_run_output: str,
     ) -> None:
         self.run(
-            f"run --name {container_name} " f"{image_name} 1> {docker_run_output} 2>&1",
+            f"run --name {container_name} {image_name} 1> {docker_run_output} 2>&1",
```

Comment:
is it "1>" is covered in "2>&1" ?

Reply:
This was just a formatting change automatically done by black. But I can take a look

Reply:
I think `run --name {container_name} {image_name} > {docker_run_output} 2>&1` should be enough

Reply:
fixed. Above is working fine